### DISCUSSION TITLE
[Bug]: 'Objects with metadata' field issue with fieldcollection

### DISCUSSION THREAD

**Discussion by sherin-shelvan-breville** [pimcore/pimcore#15657] [Q&A / Support]
### Pimcore version

6.9.6

### Steps to reproduce

https://github.com/pimcore/pimcore/issues/3558
getting the same issue in 6.9.6.  2 times notices this issue but I cannot reproduce it.  

### Actual Behavior

https://github.com/pimcore/pimcore/issues/3558
getting the same issue in 6.9.6.  2 times notices this issue but I cannot reproduce it.  

### Expected Behavior

https://github.com/pimcore/pimcore/issues/3558
getting the same issue in 6.9.6.  2 times notices this issue but I cannot reproduce it.

**Comment by dvesh3** [comment]
Hi @sherin-shelvan-breville Pimcore v6.9 is not supported anymore so moved the issue to discussions.

**Reply by dvesh3** [reply]
so you might get answer from the community for your problem.
